docs/HowTo/Configure/HighAvailability.md

```diff
@@ -1,133 +1,27 @@
 # GoQuorum high availability
 
-GoQuorum architecture allows for end to end high availability on various i/o operations to fulfill security
-and compliance requirements. In this section we will go through an example configuration and setup:
+You can configure GoQuorum for end-to-end high availability (HA). This section covers the HA
+configuration requirements for GoQuorum and Tessera.
 
-!!!warning
-    Below high availability setup is an example of how to achieve end to end high availability
-    using proxy server but it should be noted that this hasn't been tested in a production environment setup.
+![Quorum Tessera HA Mode](../../images/QT_HA_1.png)
 
-## GoQuorum Node Configuration Requirements
+## GoQuorum HA configuration requirements
 
-- Two or more GoQuorum Nodes serve as one client node.
-- The inbound RPC requests from clients will be load balanced to one of these Quorum nodes.
-- These nodes will need to share same key for transaction signing and should have shared access to key store directory or key vaults.
-- These nodes need to share the same private state. They could either connect to local Tessera node or
-    in full HA setup using [proxy](#proxy-setup-on-both-quorum-nodes) running on each GoQuorum node
-    listening on local IPC socket and directing request to Tessera Q2T HTTP but in both cases the Tessera node(s) share the same database.
+- Two or more GoQuorum nodes serve as one client node.
+- The inbound RPC requests are load balanced to one of the GoQuorum nodes.
+- The GoQuorum nodes must share the same key for transaction signing, and must have shared access to
+    the key store directory or key vaults.
+- The GoQuorum nodes must share the same private state. They can connect to a local Tessera node,
+    or a highly available Tessera node.
 
-## Tessera Node Configuration Requirements
+## Tessera HA configuration requirements
 
-- Separate [Proxy](#standalone-proxy-server-setup) server to redirect/mirror requests to two or more Tessera nodes
-- Two or more Tessera Nodes serve as Privacy manager for Client GoQuorum node.
-- These nodes share same public/private key pair (stored in password protected files or external vaults) and share same database.
-- In the server configuration, the `bindingAddress` should be the local addresses (their real addresses), but `advertisedAddress` (`serverAddress`) needs to be configured to be the proxy.
-- Add DB replication or mirroring for Tessera private data store and the JDBC connection string to include both Primary DB and DR DB connections to facilitate auto switchover on failure.
+- Two or more Tessera nodes serve as the privacy manager for a GoQuorum node.
+- The Tessera nodes share the same public and private key pair in password protected files or
+    external vaults.
+- The nodes share the same database.
```

Comment:
Can maybe remove this line and add it in the database line at the end?

Reply:
Done